deployment/docker-compose/README.md

```diff
@@ -31,6 +31,27 @@ cd deployment/docker-compose
 docker compose up -d --build
 ```
 
+## Exposing Database and Backend Ports for Local Development
+
+If you're a developer and need to expose the database and backend ports to your host machine for local testing or debugging, you can use the provided Docker Compose override file.
+
+### Steps to Expose Ports
+
+Run the following command to start the services and expose the ports:
+
+```bash
+docker compose -f docker-compose.yml -f docker-compose.override.yml up
+```
+
+This will expose the following services to the host machine
+
+- Redis: Available on port 6379
+- ClickHouse: Available on ports 8123 (HTTP) and 9000 (Native Protocol)
+- MySQL: Available on port 3306
+- Backend: Available on ports 8080 and 3003
```

Comment:
Minor: you could clarify that 8080 is for REST endpoints and 3003 for the Open API specification.